code/backend/batch/AddURLEmbeddings.py

```diff
@@ -17,34 +22,54 @@ def add_url_embeddings(req: func.HttpRequest) -> func.HttpResponse:
     logger.info("Python HTTP trigger function processed a request.")
 
     # Get Url from request
-    url = req.params.get("url")
-    if not url:
-        try:
-            req_body = req.get_json()
-        except ValueError:
-            pass
-        else:
-            url = req_body.get("url")
-    # Check if url is present, compute embeddings and add them to VectorStore
-    if url:
-        try:
-            config = ConfigHelper.get_active_config_or_default()
-            document_processor = DocumentProcessor()
-            processors = list(
-                filter(lambda x: x.document_type == "url", config.document_processors)
-            )
-            document_processor.process(source_url=url, processors=processors)
-        except Exception:
-            return func.HttpResponse(
-                f"Error: {traceback.format_exc()}", status_code=500
-            )
+    url = None
+    try:
+        req_body = req.get_json()
+    except ValueError:
+        pass
+    else:
+        url = req_body.get("url")
```

Comment:
can we write this as a 1 liner, saves from 3 lines of code-voodoo with exception block.

```suggestion
url = req.get_json().get("url") if req.get_json() else None 
```

Reply:
This could still throw an exception